pkgs/servers/tailscale/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "tailscale";
+  version = "0.96-33";
+
+  src = fetchFromGitHub {
+    owner = "tailscale";
+    repo = "tailscale";
+    rev = "19cc4f8b8ecfdc16136d8489a1c2b899f556fda7";
+    sha256 = "0kcf3mz7fs15dm1dnkvrmdkm3agrl1zlg9ngb7cwfmvkkw1rkl6i";
+  };
+
```

Comment:
I forgot to ask for this earlier, but could you set CGO_ENABLED=0 here? (between the src= and goPackagePath blocks)

We disable cgo in all Tailscale binaries, to reduce external dependencies and because, as a VPN system, we need fine-grained control over DNS resolution that the libc resolver cannot give us. I don't _think_ enabling cgo would break anything right now, but it's likely to break stuff in the future, and I'd like the Nix derivation to be on-par with the rest of Tailscale's package builds.

Reply:
Makes sense :+1: 

Updated the PR.

Before:
```
λ NIX_PATH=nixpkgs=/home/martin/Code/personal/nixpkgs nix-shell -p tailscale --run 'ldd `which tailscale`'
        linux-vdso.so.1 (0x00007ffe00ffd000)
        libpthread.so.0 => /nix/store/4552zjr164h964cxq8iw5zgnlcx1vwbl-glibc-2.30/lib/libpthread.so.0 (0x00007f332e8ae000)
        libc.so.6 => /nix/store/4552zjr164h964cxq8iw5zgnlcx1vwbl-glibc-2.30/lib/libc.so.6 (0x00007f332e6ef000)
        /nix/store/4552zjr164h964cxq8iw5zgnlcx1vwbl-glibc-2.30/lib/ld-linux-x86-64.so.2 => /nix/store/4552zjr164h964cxq8iw5zgnlcx1vwbl-glibc-2.30/lib64/ld-linux-x86-64.so.2 (0x00007f332e8d1000)
```

After:
```
λ NIX_PATH=nixpkgs=/home/martin/Code/personal/nixpkgs nix-shell -p tailscale --run 'ldd `which tailscale`'
        not a dynamic executable
```